package.json

```diff
@@ -49,7 +49,7 @@
     "moment": "^2.29.3",
     "moment-timezone": "^0.5.34",
     "multer": "^1.4.5-lts.1",
-    "next": "^13.4.4",
+    "next": "13.1.1",
```

Comment:
Can we keep the version to 13.4.4?

Reply:
I've been experiencing font issues if the next versions > 13.1.1. I researched  over the internet and people are also experiencing the same issue. https://github.com/vercel/next.js/issues/45052 

This is a sample screenshot if the version is > 13.1.1

<img width="1512" alt="Screenshot 2023-06-09 at 8 59 51 AM" src="https://github.com/klaudsol/klaudsol-cms/assets/101687552/8d91a531-0501-4986-b4d6-3c019628f261">

My node version is 20.3.0. I also tried 16.18.0 and 18.13.0, but the result is the same if the version is 13.4.4.